sdks/python/apache_beam/typehints/native_type_compatibility_test.py

```diff
@@ -38,9 +38,12 @@ class _TestClass(object):
 
 
 class NativeTypeCompatibilityTest(unittest.TestCase):
-  @unittest.skipIf(sys.version_info[0] == 3 and
+
+  @unittest.skipIf(sys.version_info[0] == 3 and sys.version_info[1] < 6 and
```

Comment:
sys.version_info[1] < 3?

Reply:
Changed to `[sys.hexversion](https://docs.python.org/3/library/sys.html#sys.hexversion)` for easy checking.